Added mention of webpack-flush-chunks-html plugin.

m4r1vs opened the pull request:
Now mentioning the plugin in readme under the "How do I serve the CSS" section. For more information about the plugin see its GitHub repo: https://github.com/m4r1vs/webpack-flush-chunks-html :)
 
Btw, thanks for this awesome plugin, haven't had any problems yet and transitioning was smooth af 👍

faceyspacey commented:
glad to hear! thanks for this.

...one thing real quick, do you mind making one more quick (perhaps an empty line) with the commit script: `yarn run cm`. That way it will auto publish to NPM when I merge it (it formats your commits in a special way).

m4r1vs commented:
Did so, also added a missing '<' to the author field of package.json: 48af2b628e2e9df6f0bbb7a5c2ec2f617dc0bf77. Running the command also modified yarn.lock, seems a bit weird but shouldn't break anything. If you think I should revert that change, let me know tho :)

faceyspacey commented:
all good thanks.
